### PR TITLE
Add Weaver TestModule

### PR DESCRIPTION
**Pull request by lolgab**
Weaver is a test framework developed by Disney Streaming:
https://github.com/disneystreaming/weaver-test

This PR adds a `TestModule.Weaver` convenience trait.

### CODE REVIEW

**Review by lefou — COMMENTED** [review]
I think adding a link to the project providing that test framework would be nice.

**Comment by lolgab** [comment]
> I think adding a link to the project providing that test framework would be nice.

To the PR description/commit or to the code?

**Comment by lefou** [comment]
> > I think adding a link to the project providing that test framework would be nice.
> 
> To the PR description/commit or to the code?

ScalaDoc would be the most helpful, I guess.

File: scalalib/src/TestModule.scala [hunk 1]
```diff
@@ -233,6 +233,14 @@ object TestModule {
     override def testFramework: T[String] = "munit.Framework"
   }
 
+  /**
+   * TestModule that uses MUnit to run tests.
+   * You need to provide the weaver dependencies yourself.
+   */
```
Comment (lefou) [inline scalalib/src/TestModule.scala:239]:
```suggestion
  /**
   * TestModule that uses MUnit to run tests.
   * You need to provide the weaver dependencies yourself.
   * https://github.com/disneystreaming/weaver-test
   */
```

**Pull request merged**